server/views/pages/induction/affectAbilityToWork/index.njk

```diff
@@ -0,0 +1,122 @@
+{% extends "../../../partials/layout.njk" %}
+
+{% set pageId = "induction-affect-ability-to-work" %}
+{% set title = "Is there anything that " + prisonerSummary.firstName + " " + prisonerSummary.lastName + " feels may affect their ability to work after they're released?" %}
+{% set pageTitle = title %}
+
+{#
+Data supplied to this template:
+  * prisonerSummary - object with firstName and lastName
+  * backLinkUrl - url of the back link
+  * backLinkAriaText - the aria label for the back link
+  * form - form object containing the following fields:
+    * affectAbilityToWork - Array existing values for field
+    * affectAbilityToWorkOther? - value for when Other is selected
+  * errors? - validation errors
+#}
+
+{% block beforeContent %}
+  {{ super() }}
+  {{ govukBackLink({ text: "Back", href: backLinkUrl, attributes: { "aria-label" : backLinkAriaText } }) }}
+{% endblock %}
+
+{% block content %}
+
+  {% if errors.length > 0 %}
+    {{ govukErrorSummary({
+      titleText: 'There is a problem',
+      errorList: errors,
+      attributes: { 'data-qa-errors': true }
+    }) }}
+  {% endif %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <form class="form" method="post" novalidate="">
+        <div class="govuk-form-group">
+          <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+
+          {% set otherHtml %}
+            {{ govukTextarea({
+              id: "affectAbilityToWorkOther",
+              name: "affectAbilityToWorkOther",
+              rows: "2",
+              value: affectAbilityToWorkOther,
+              type: "text",
+              label: {
+                text: "Give details",
+                attributes: { "aria-live": "polite" }
+              },
+              attributes: { "aria-label" : "Give details of any affectAbilityToWork" },
+              errorMessage: errors.affectAbilityToWorkOther
+            }) }}
+          {% endset -%}
+
+          {{ govukCheckboxes({
+            name: "affectAbilityToWork",
+            fieldset: {
+              legend: {
+                  text: title,
+                  isPageHeading: true,
+                  classes: "govuk-fieldset__legend--l"
+                }
+            },
+            hint: {
+              text: "Select all that apply."
+            },
+            items: [
+              {
+                value: "LIMITED_BY_OFFENSE",
+                checked: form.affectAbilityToWork.includes("LIMITED_BY_OFFENSE"),
+                text: "LIMITED_BY_OFFENSE" | formatAffectAbilityToWork
```

Comment:
The filter for this already exists, and is called `formatAbilityToWorkConstraint`
```
text: "LIMITED_BY_OFFENSE" | formatAbilityToWorkConstraint
```
(same for all the other checkbox options)

Reply:
Ah yes, that's good (even if I prefer `formatAffectAbilityToWork` 😂)